src/components/Theme/src/default-theme.js

```diff
@@ -41,19 +44,17 @@ export default function defaultTheme() {
 		text: {
 			fontFamily: 'inherit',
 			size: 0,
-			textColor: undefined,
 		},
 		heading: {
 			fontFamily: 'inherit',
 			size: 2,
-			textColor: undefined,
```

Comment:
If we set this and `text` to `textColor: '@colors.heading'` and `textColor: '@colors.text'`, we won't need to create additional variables for `--color-text` and `--color-heading`.  I also like showing undefined values in this default theme because it makes clear what all the themable props are for each themeable component.

Reply:
This was intentional, but may be slightly subversive. The colors for heading, text, and other color tokens that are part of our color profiles aren't always a one-to-one relationship between the token and the component. Global generic color tokens should be reusable.

For example, the Calendar component has a header that does not use the Heading component. The text color of our inline forms should inherit the text color, but they aren't using the text component.

So an opinionated approach here is rather than explicitly declaring the defaults on components in the theme definition we should look at the theme definition as setting global reusable tokens. This will be something I'll push here and keep on pushing, but this was my gentle nudge regarding this direction.

Considering future scenario we will need to build, the "shape" token that should define the visual style of more than just the button. Given our current approach we have three distinct and named shapes in the button component. The implication is that we will duplicate these three shapes in every component. Then the theme definition will have a declaration for every component to then set the same prop on every component. It's an incredible amount of extra code that isn't necessary and would be better served through a single prop like borderRadius that defaults to a global token(s). This is the pattern established in other themable design systems like Material and defined in the Theme UI spec as `radii`.  https://github.com/square/maker/blob/ba223d1a518cc156d265d79715369d04df4ae3fa/lab/experiments/Theme/themes.js#L64-L66

This simple generic definition is far easier to understand, more flexible, and provides far less of a footprint for us and others to have to manage as we expand on our theme definitions.